.github/workflows/pull-request.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         run: |
           cargo fmt --check
 
+      - name: Install protoc for the gRPC extension
```

Comment:
We should start using what-rust-changed in this repo and make this step conditional on the gRPC extension having changed